controller/internal/exporterset/provisioners/cuttlefish/cuttlefish.go

```
@@ -0,0 +1,480 @@
+	if exporter != nil {
+		exporterContainer.Env = append(exporterContainer.Env, corev1.EnvVar{
+			Name:  "JUMPSTARTER_EXEC_LOG_FIELDS",
+			Value: fmt.Sprintf("component=exporter,exporter=%s,namespace=%s", exporter.Name, exporter.Namespace),
+		})
+	}
+
+	imageVolume := corev1.Volume{
+		Name: "cvd-images",
+	}
+	if imageClaim != "" {
+		imageVolume.VolumeSource.PersistentVolumeClaim = &corev1.PersistentVolumeClaimVolumeSource{
```

golangci-lint (2.13.2)

[error] QF1008: could remove embedded field "VolumeSource" from selector

(staticcheck)